packages/configuration-validation/package.json

```diff
@@ -35,8 +35,10 @@
     "babel-cli": "^6.26.0",
     "babel-eslint": "^8.1.1",
     "babel-preset-env": "^1.7.0",
+    "deep-extend": "^0.6.0",
     "eslint": "^4.7.1",
-    "jest": "^23.6.0"
+    "jest": "^23.6.0",
+    "lodash": "^4.17.15"
```

Comment:
Note for anyone curious: This lodash version addresses the known security issues.